go/client/cmd_verify.go

```diff
@@ -113,19 +121,24 @@ func (c *CmdVerify) Run() (err error) {
 		return err
 	}
 	snk, src, err := c.ClientFilterOpen(c.G())
-	if err == nil {
-		arg := keybase1.SaltpackVerifyArg{
-			Source: src,
-			Sink:   snk,
-			Opts: keybase1.SaltpackVerifyOptions{
-				Signature: c.detachedData,
-				SignedBy:  c.signedBy,
-			},
-		}
-		err = cli.SaltpackVerify(context.TODO(), arg)
+	if err != nil {
+		return err
 	}
-	cerr := c.Close(err)
-	return libkb.PickFirstError(err, cerr)
+
+	arg := keybase1.SaltpackVerifyArg{
+		Source: src,
+		Sink:   snk,
+		Opts: keybase1.SaltpackVerifyOptions{
+			Signature: c.detachedData,
+			SignedBy:  c.signedBy,
+		},
+	}
+	err = cli.SaltpackVerify(context.TODO(), arg)
+	if err != nil {
+		return err
+	}
+
+	return c.Close(err)
```

Comment:
Ack you're right, I thought I was cleaning this up when I didn't actually understand it.